6-AdvancedScenarios/4-sign-in-hybrid/README.md

```diff
@@ -0,0 +1,357 @@
+1. Select **Save** to save your changes.
+1. In the app's registration screen, select the **Certificates & secrets** blade in the left to open the page where you can generate secrets and upload certificates.
+1. In the **Client secrets** section, select **New client secret**:
+   * Type a key description (for instance `app secret`),
+   * Select one of the available key durations (**6 months**, **12 months** or **Custom**) as per your security posture.
+   * The generated key value will be displayed when you select the **Add** button. Copy and save the generated value for use in later steps.
+   * You'll need this key later in your code's configuration files. This key value will not be displayed again, and is not retrievable by any other means, so make sure to note it from the Azure portal before navigating to any other screen or blade.
+1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
+      * Select the **Add a permission** button and then:
+      * Ensure that the **Microsoft APIs** tab is selected.
+      * In the *Commonly used Microsoft APIs* section, select **Microsoft Graph**
+      * In the **Delegated permissions** section, select the **User.Read** in the list. Use the search box if necessary.
+      * Select the **Add permissions** button at the bottom.
+1. Still on the same app registration, select the **Token configuration** blade to the left.
+1. Select **Add optional claim**:
+    * Select optional claim type, choose **ID**.
+    * Select optional claim name, choose **sid**.
+    * Select optional claim name, choose **login_hint**.
+1. Select **Add** to save changes.
+
+#### Configure the app (msal-hybrid-spa) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `App/.env` file.
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-hybrid-spa` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant ID.
+1. Find the key `Enter_the_Client_Secret_Here` and replace the existing value with the key you saved during the creation of `msal-hybrid-spa` copied from the Azure portal.
+
+In addition:
+
+1. Open the `App/client/authConfig.js` file.
+1. Find the string `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-hybrid-spa` app copied
+1. Find the string `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant ID.
+
+### Step 4: Running the sample
+
+For command line run the next commands:
+
+* Run the app:
+
+  ```console
+    cd 6-AdvancedScenarios\4-sign-in-hybrid\App
+    npm start
+    ```
+
+## About the code
```

Comment:
"Explore the sample section" is missing altogether. you know:
1. Open the bowser and go to http:////<wherever>
2.
3.